### PR TITLE
[Workers] Adds New Python Worker Examples

### PR DESCRIPTION
**Pull request by megaconfidence**
Adds new Python Worker examples for:
- Cache using fetch
- Conditional response
- Cookie parsing

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]
General formatting + imports / usage looks generally correct to me.

cc: @hoodmane / @KianNH, you see anything to call out here?

**Review by irvinebroque — APPROVED** [review]

File: content/workers/examples/cache-using-fetch.md [hunk 1]
```diff
@@ -72,6 +73,41 @@ export default {
...
+    some_custom_key = f'https://{url.hostname}{url.pathname}'
+
+    response = await fetch(request, to_js({"cf": {
+        # Always cache this fetch regardless of content type
+        # for a max of 5 seconds before revalidating the resource
+        "cacheTtl": 5,
+        "cacheEverything": True,
+        # Enterprise only feature, see Cache API for other plans
+        "cacheKey": some_custom_key,
+        }}))
```
Comment (hoodmane) [inline content/workers/examples/cache-using-fetch.md]: Maybe 
```
cf=to_js({
   "cacheTtl": 5,
   ...
})
```

File: content/workers/examples/cache-using-fetch.md [hunk 2]
```diff
@@ -72,6 +73,41 @@ export default {
...
+
+    # Reconstruct the Response object to make its headers mutable
+    response = Response.new(response.body, response)
+
+    # Set cache control headers to cache on browser for 25 minutes
+    response.headers.set("Cache-Control", "max-age=1500")
```
Comment (hoodmane) [inline content/workers/examples/cache-using-fetch.md]:
```suggestion
    response.headers["Cache-Control"] = "max-age=1500"
```

File: content/workers/examples/extract-cookie-value.md [hunk 3]
```diff
@@ -51,6 +52,28 @@ export default {
...
+    # Name of the cookie
+    cookie_name = "__uid"
+
+    cookies = SimpleCookie()
+    cookies.load(request.headers["Cookie"] or "")
+    cookies = {k: v.value for k, v in cookies.items()}
+
+    if cookie_name in cookies:
+        # Respond with cookie value
+        return Response.new(cookies[cookie_name])
```
Comment (hoodmane) [inline content/workers/examples/extract-cookie-value.md]:
```suggestion

    if cookie_name in cookies:
        # Respond with cookie value
        return Response.new(cookies[cookie_name].value)
```

**Review by hoodmane — APPROVED** [review]
Thanks @megaconfidence! I have some minor nitpicks as usual.

File: content/workers/examples/extract-cookie-value.md [hunk 4]
```diff
@@ -51,6 +52,28 @@ export default {
...
+```py
+from http.cookies import SimpleCookie
+from js import Response
+
+async def on_fetch(request):
+    # Name of the cookie
+    cookie_name = "__uid"
+
+    cookies = SimpleCookie()
+    cookies.load(request.headers["Cookie"] or "")
```
Comment (hoodmane) [inline content/workers/examples/extract-cookie-value.md]: I think you can do:
```suggestion
    cookies = SimpleCookie(request.headers["Cookie"] or "")
```

File: content/workers/examples/conditional-response.md [hunk 5]
```diff
@@ -113,5 +114,52 @@ export default {
...
+    # Block on hostname
+    if url.hostname in blocked_hostnames:
+        return Response.new("Blocked Host", status=403)
+
+    # On paths ending in .doc or .xml
+    if re.search(r'\.(doc|xml)$', url.pathname):
+        return Response.new("Blocked Extension", status=403)
+
+    # On HTTP method
+    if "POST" in request.method:
```
Comment (garrettgu10) [inline content/workers/examples/conditional-response.md:137]: Should this be `request.method == "POST"`?

**Pull request merged**